src/components/add-journey/JourneySearchResults.tsx

```diff
@@ -30,11 +36,47 @@ const ResultDisplay: React.FC = () => {
   }
 
   return (
-    <ul role="list">
-      {journeys.map((journey) => (
-        <JourneySearchResult key={generateJourneyKey(journey)} journey={journey} />
-      ))}
-    </ul>
+    <>
+      <button
+        disabled={isFetching}
+        onClick={() => {
+          // Subtracts 2 hours to the current departureTime
+          const newDepartureTime = new Date(departureTime);
+          newDepartureTime.setHours(newDepartureTime.getHours() - 2);
```

Comment:
For now I would move this to -0.5 hours, as we run the risk of not covering all connections otherwise.

And since we use `date-fns` in a lot of places in this code base, I would use it here as well: `subHours`

Reply:
ah got it, wasn't sure what was available here. Sounds good I'll make these changes later today